### ISSUE TITLE
Ensure DSC Configuration on broker is only applied once

### ISSUE THREAD

**Issue by ghost** [mmarch/rds-templates#31]
Error encountered when shutting down and restarting VMs where DSC Configuration tried to run again.

[RE Quickstart template behavior questions.txt](https://github.com/mmarch/rds-templates/files/797437/RE.Quickstart.template.behavior.questions.txt)

**Comment by mmarch** [comment]
I've verified that our DCS configuration at 
https://github.com/Azure/azure-quickstart-templates/tree/master/rds-deployment/Configuration.zip | Confgiration/Configuration.ps1
does have the LCM setting for ConfigurationMode, and it's set to "[ApplyOnly](https://msdn.microsoft.com/en-us/powershell/dsc/metaconfig)", which is supposed to stop doing any changes after applying initial configuration:
![image](https://cloud.githubusercontent.com/assets/11985571/23876486/f46f6a64-07fa-11e7-9729-4a7b69771001.png)

however, this apparently doesn't work as intended. We'll need to follow up with DSC extension owners re this behavior.

**Comment by mmarch** [comment]
@Haley-Rowland Haley, do you still have that deployment available by any chance? I wanted pull DSC logs from the Broker to send to DSC extension dev to look at...

**Comment by ghost** [comment]
Yes, it's grprimary in RDS_AZURE_REFARCH_003. Just made you a co-admin to get access.

**Comment by mmarch** [comment]
After looking at DSC logs I found that DSC config was applied twice by two different versions of DSC extension handler (ver. 2.21.0.0 on 1/27, and ver. 2.22.0.0 once it was released on 2/21); each time DSC config was applied once, as expected.  
What's "not expected" is DSC extension handler newer version not aware of the fact the config already was applied before by older version.

This is a result of us having "autoUpgradeMinorVersion" set to "true"  in our templates:
![image](https://cloud.githubusercontent.com/assets/11985571/24065598/854b07c4-0b29-11e7-9a86-a01d8933efd1.png)

Closing this as external issue.  I'll try to follow up with DSC extension owners re this scenario.
